design/features/routing/version-1.md

```diff
@@ -28,57 +28,57 @@ ___
 
 ### As-is
 
-- form creator creates questions and marks them as optional giving content as part of hint text to explain who needs complete a question
+- Form creator creates questions and marks them as optional. Form creators can use hint text to explain who needs to complete a question.
 
 ### To-be
 
-- from creator can select radio option and set a page to go to if selected. If a radio option has no set page then option will go to next page in the order (unless another option has the next page set as the next step)  
-- form creator HAS to select a next question for each radio option IF one radio option in a list has had a next question set  
-- form creator can set next (single) question as skippable if radio option X is chosen  
+- Form creator can create a 'question route' by selecting a radio option ('select from a list' question) and setting a page to go to depending on the answer to the route's start question. If a radio option has no set page then option will go to next page in the order (unless another option has the next page set as the next step)  
+- Form creator **must** select a next question for each radio option **if** one radio option in a list has had a next question set  
+- Form creator can set next (single) question as skippable if radio option X is chosen  
 
 
 ## Key decisions
 
-- we are only going to allow routing to be based on a radio button answer type
-- there can only be one route for a question
-  - a question can be routed to and then routed from
-  - but cannot have multiple routes from
-- we will not be considering branching journeys as part of this work
-- we will prioritise errors for breaking changes, such as
-  - the answer to base the route on is changed or deleted
+- We're only going to allow routing to be set up from a radio button answer type 
+- Only one route can be set up for each question
+  - A question can be routed to and then routed from
+  - A question cannot have multiple routes from it
```

Comment:
Is this bullet point not repeating the main bullet ("Only one route can be set up for each question")?

Reply:
Yeah, it does I think I wrote it to clarify when it came to development as part of the conversations we were having. 
Happy to remove or keep